argparse.c

```diff
@@ -384,3 +384,12 @@ argparse_help_cb(struct argparse *self, const struct argparse_option *option)
     argparse_usage(self);
```

Comment:
```suggestion
argparse_help_cb_no_exit(self, option);
```
Just to avoid the code duplication and make both versions to stay in sync.

Reply:
PS: I did select two lines before posting the suggestion but it seems GitHub doesn't support this (contrary to [this announcement](https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/)). So please make sure to also drop the `void(option);` line from `argparse_help_cb` when applying this change.

Reply:
I decided to split it into two separate functions to maintain backwards compatibility.

Reply:
Oh, I forgot to put it in the header file. Oops...

Reply:
I don't understand that point: If `argparse_help_cb_no_exit` is identical to `argparse_help_cb` expect for the last line then how would calling it there break backwards compatibilty. Clearly I miss something here because I don't get the point of introducing `argparse_help_cb_no_exit` without using or exporting it in the first place.

Reply:
> I don't understand that point: If `argparse_help_cb_no_exit` is identical to `argparse_help_cb` expect for the last line then how would calling it there break backwards compatibilty? Clearly I miss something here because I don't get the point of introducing `argparse_help_cb_no_exit` without using or exporting it in the first place.

Existing code would likely be broken by the new argument if we were not to duplicate the function.

Reply:
What new argument?

Do make it clear, I suggest replacing

```c
int
argparse_help_cb(struct argparse *self, const struct argparse_option *option)
{
    (void)option;
    argparse_usage(self);
    exit(EXIT_SUCCESS);
}

int
argparse_help_cb_no_exit(struct argparse *self,
                         const struct argparse_option *option)
{
    (void)option;
    argparse_usage(self);
}
```

by

```c
int
argparse_help_cb_no_exit(struct argparse *self,
                         const struct argparse_option *option)
{
    (void)option;
    argparse_usage(self);
}

int
argparse_help_cb(struct argparse *self, const struct argparse_option *option)
{
    argparse_help_cb_no_exit(self, option)
    exit(EXIT_SUCCESS);
}
```

Reply:
LGTM.. Let me `git rebase -i` real quick.